sdk/storage/Azure.Storage.Blobs/tests/BlobUriBuilderTests.cs

```diff
@@ -220,24 +220,28 @@ public void BlobUriBuilder_RegularUrl_CNAME()
         }
 
         [Test]
-        public void BlobUriBuilder_RegularUrl_MalformedSubdomain()
+        public void DataLakeUriBuilder_AccountNamePeriod()
```

Comment:
Could you add test that we handle this case as well? https://github.com/Azure/azure-sdk-for-java/blob/5672b860b15aa32d1dd5b559f47c3cb1058f0166/sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobUrlParts.java#L435-L436

We should also check that "azurite" urls parse correctly. Could you check if we have coverage like this in .NET https://github.com/Azure/azure-sdk-for-java/blob/5672b860b15aa32d1dd5b559f47c3cb1058f0166/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AzuriteTest.groovy#L48-L74 ? If not could you add it?

Reply:
Should be done. There were also some existing tests that covered some of these cases.